build/moz.configure/bindgen.configure

```diff
@@ -12,7 +12,7 @@ option(env="CBINDGEN", nargs=1, help="Path to cbindgen")
 def check_cbindgen_version(cbindgen, fatal=False):
     log.debug("trying cbindgen: %s" % cbindgen)
 
-    cbindgen_min_version = Version("0.26.0")
+    cbindgen_min_version = Version("0.24.3")
```

Comment:
Revert this change we need cbindgen originally specified. Otherwise encounter this error.

https://github.com/kenz-gelsoft/gecko-dev/issues/10#issuecomment-2267273031